.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,67 +1,44 @@
 ---
-name: Bug report
+name: Library-specific Bug Report
 about: Report a problem with this library that needs to be fixed.
 title: ''
 labels: bug, triage
 assignees: ''
 
 ---
-<!--
-**PLEASE READ**
-==============================================================================
-Before posting, make sure you've carefully reviewed the documentation for this
-library:
-
-https://developers.google.com/google-ads/api/docs/client-libs/python
-
-This library is only used for the Google Ads API. For the AdWords API client
-library report your issue here:
-
-https://github.com/googleads/googleads-python-lib/issues
-
-Only use this page to report problems with this library, which is designed for
-the Python language.
+<!-- PLEASE READ
+This form is for bug reports specific to this library.
 
-If you are looking for help with a different language, see:
+For troubleshooting tips, see: https://developers.google.com/google-ads/api/docs/best-practices/troubleshooting
 
-https://developers.google.com/google-ads/api/docs/client-libs
+For issues that are related to the Google Ads API itself and not specific to this library, please reach out to one of our support channels at https://developers.google.com/google-ads/api/support.
```

Comment:
Can you add a colon to the end of this line.

Reply:
Done